test/epiviewpoint/data_file_test.exs

```diff
@@ -197,5 +197,134 @@ defmodule EpiViewpoint.DataFileTest do
          ]}
       )
     end
+
+    test "reads a bulk FHIR file" do
+      input = [
+        %{
+          "first_name" => "Alice",
+          "last_name" => "Ant",
+          "dob" => "01/02/1970",
+          "sample_date" => "06/01/2020",
+          "result_date" => "06/03/2020",
+          "result" => "positive"
+        },
+        %{
+          "first_name" => "Billy",
+          "last_name" => "Bat",
+          "dob" => "03/04/1990",
+          "sample_date" => "06/06/2020",
+          "result_date" => "06/07/2020",
+          "result" => "negative"
+        }
+      ]
+
+      assert {:ok, result} =
```

Comment:
NBD but you don't need this `assert` if you really want to test the `result` later, which I believe you do.